pkg/local_object_storage/shard/mode.go

```diff
@@ -68,3 +71,8 @@ func (s *Shard) GetMode() mode.Mode {
 
 	return s.info.Mode
 }
+
+// IsReadOnly return true if shard not in mode ReadWrite.
+func (s *Shard) IsReadOnly() bool {
```

Comment:
First, we should take a mutex, second we already have `GetMode`, I think it solves the job.

Reply:
Refactored.